test/hotspot/jtreg/containers/docker/TestMemoryAwareness.java

```diff
@@ -150,6 +158,12 @@ private static void testOperatingSystemMXBeanAwareness(String memoryAllocation,
             .addJavaOpts("--add-exports")
             .addJavaOpts("java.base/jdk.internal.platform=ALL-UNNAMED");
 
+        if (!isUseSwap) {
+            opts.addDockerOpts("--memory-swappiness", "0");
+        } else {
+            opts.addDockerOpts("--memory-swappiness", "60");
```

Comment:
Unfortunately this breaks on a cgroups v2 system as `--memory-swappiness` is not supported there. I'd prefer if this wouldn't piggy back on the existing test, but actually assert that swap is properly reported as the same as the memory limit if `--memory-swappiness=0`. Also, this test only verifies the Java (core-libs) change, not the hotspot change. That would have to be done via some `TestMisc` variant which uses `print_container_info()`.